### PR TITLE
Feat add comments to define tables and fields

### LINKED ISSUE #5: Comment tables and fields when creating migrations
Now with Rails 5 we can add comments on tables and their fields.

These comments are important so everyone can understand what information the table holds and what each field represents.

This can be done like this:
```ruby
# frozen_string_literal: true

class ExampleTable < ActiveRecord::Migration[5.0]
  def change
    create_table :example_table, comment: 'The is an example table for this suggestion' do |t|
      t.references :example_reference, index: false, foreign_key: true, comment: 'A reference example'

      t.integer :example_field, comment: 'The is an example field for this table'

      t.timestamps null: false
    end
  end
end

```

### PR DESCRIPTION
**Pull request by renanrcp**
Add Rails 5+ comments approach to define tables and fields on backend styleguide

Closes #5

### CODE REVIEW

File: backend-styleguide/README.md [hunk 1]
```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
 ], scope: true
 ```
 
+### 👉 Use comments to define tables and fields
+
+We use comments to define tables and fields. This is a good approach to help newcomers and others Geeks.
```
Comment (pedro-lb) [inline backend-styleguide/README.md:318]:
```suggestion
We use comments to define tables and fields. This is a good approach to help newcomers and other Geeks.
```

File: backend-styleguide/README.md [hunk 2]
```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
...
+
+Don't put sensitive information, comments doesn't have any security mechanism and they are stored globally so any user connected to any database in cluster can see all comments for shared objects.
```
Comment (pedro-lb) [inline backend-styleguide/README.md:320]:
```suggestion
Don't put sensitive information, comments don't have any security mechanism and they are stored globally so any user connected to any database in the cluster can see all comments for shared objects.
```

File: backend-styleguide/README.md [hunk 3]
```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
...
+
+Add rich information to the comment, just write the name of the field or table don't show what it is.
```
Comment (pedro-lb) [inline backend-styleguide/README.md:322]: I think we can make this clearer:

```suggestion
Avoid adding obvious information. Good practices on code commenting also apply here!
```

File: backend-styleguide/README.md [hunk 4]
```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
...
+# frozen_string_literal: true
+
+class Person < ActiveRecord::Migration[5.0]
+  def change
+    create_table :person, comment: 'Stores primary person data' do |t|
+      t.references :address, index: false, foreign_key: true, comment: 'A reference for the address of this people'
+
+      t.string :phone_number, comment: 'The phone number of this people including country and area code'
+
+      t.string :name, comment: 'The complete name of this people'
```
Comment (pedro-lb) [inline backend-styleguide/README.md:355]:
```suggestion
      t.string :name, comment: 'The complete name of this person'
```

File: backend-styleguide/README.md [hunk 5]
```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
...
+✅ Good
+```ruby
+# frozen_string_literal: true
+
+class Person < ActiveRecord::Migration[5.0]
+  def change
+    create_table :person, comment: 'Stores primary person data' do |t|
+      t.references :address, index: false, foreign_key: true, comment: 'A reference for the address of this people'
+
+      t.string :phone_number, comment: 'The phone number of this people including country and area code'
```
Comment (pedro-lb) [inline backend-styleguide/README.md:353]:
```suggestion
      t.string :phone_number, comment: 'The phone number of this person including country and area code'
```

File: backend-styleguide/README.md [hunk 6]
```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
...
+```
+
+✅ Good
+```ruby
+# frozen_string_literal: true
+
+class Person < ActiveRecord::Migration[5.0]
+  def change
+    create_table :person, comment: 'Stores primary person data' do |t|
+      t.references :address, index: false, foreign_key: true, comment: 'A reference for the address of this people'
```
Comment (pedro-lb) [inline backend-styleguide/README.md:351]:
```suggestion
      t.references :address, index: false, foreign_key: true, comment: 'A reference for the address of this person'
```

**Review by pedro-lb — CHANGES_REQUESTED** [review]

File: backend-styleguide/README.md (hunk 6 above)
Comment (renanrcp) [reply backend-styleguide/README.md]: fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66

File: backend-styleguide/README.md (hunk 5 above)
Comment (renanrcp) [reply backend-styleguide/README.md]: fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66

File: backend-styleguide/README.md (hunk 4 above)
Comment (renanrcp) [reply backend-styleguide/README.md]: fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66

File: backend-styleguide/README.md (hunk 3 above)
Comment (renanrcp) [reply backend-styleguide/README.md]: fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66

File: backend-styleguide/README.md (hunk 2 above)
Comment (renanrcp) [reply backend-styleguide/README.md]: fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66

File: backend-styleguide/README.md (hunk 1 above)
Comment (renanrcp) [reply backend-styleguide/README.md]: fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66

**Review by paulohenriquepm — APPROVED** [review]

**Review by pedro-lb — APPROVED** [review]

**Pull request merged**